com.amphi.music.yaml

```diff
@@ -0,0 +1,67 @@
+id: com.amphi.music
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: music
+
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/bin/lib:$LD_LIBRARY_PATH
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.portal.Desktop
+  - --socket=pulseaudio 
+
+modules:
+
+  - name: music
+    buildsystem: simple
+    build-commands:
+      - mkdir -p external
+      - tar --use-compress-program=unzstd -xf music.tar.zst
+      - tar --use-compress-program=unzstd -xf external.tar.zst -C external
+      - install -Dm755 music /app/bin/music
+      - cp -r lib /app/bin/lib
```

Comment:
why in `/app/bin/lib`. This is NOT standard. The prefix is `/app` not `/app/bin`